track.go

```diff
@@ -63,56 +64,48 @@ func (t *Track) Label() string {
 	return t.label
 }
 
-// SSRC gets the SSRC of the track
-func (t *Track) SSRC() uint32 {
+// Streams return the track streams
+func (t *Track) Streams() []*TrackRTPStream {
 	t.mu.RLock()
 	defer t.mu.RUnlock()
-	return t.ssrc
+	return append([]*TrackRTPStream{}, t.streams...)
```

Comment:
Pedantic perf... costs a few more lines of code, but a `make([]*TrackRTPStream{}, len(t.streams))` and a `copy` are clearer about capacity

Reply:
I agree. Updated.